src/dmd/astbase.d

```diff
@@ -6236,7 +6230,7 @@ struct ASTBase
             const(char)* id;
         }
 
-        static __gshared SCstring* table =
+        static __gshared SCstring[31] table =
```

Comment:
Why not `static immutable` and letting the compiler infer the size?

Reply:
Yes, done.